ProjectWork/Pages/Authentication/AccountEdit.razor

```diff
@@ -13,91 +16,85 @@
+                <li>
+                    <input @bind-value="_editUser.Email" 
+                        class="w3-round w3-margin-right w-50 w3-margin-bottom" type="text" id="mail">
+                </li>
+                <li>
+                    <p class="w3-bold">Bio:</p>
                 </li>
-
                 <li>
-                    <p class="w3-bold">New password:</p>
+                <InputTextArea id="w3review" name="w3review" rows="4" cols="50" @bind-Value="_editUser.Biography" />
+                </li>
+
+                 <li>
+                    <p class="w3-bold">First Name:</p>
                 </li>
                 <li>
-                    <input class="w3-round w3-margin-right w-50 w3-margin-bottom" type="password" id="newpw">
+                    <input @bind-value="_editUser.FirstName" 
+                        class="w3-round w3-margin-right w-50 w3-margin-bottom" type="text" id="mail">
                 </li>
 
+                 <li>
+                    <p class="w3-bold">Last Name:</p>
+                </li>
                 <li>
-                    <p class="w3-bold">Repeat new password:</p>
+                    <input @bind-value="_editUser.LastName" 
+                        class="w3-round w3-margin-right w-50 w3-margin-bottom" type="text" id="mail">
+                </li>
+
+                 <li>
+                    <p class="w3-bold">Profile Picture:</p>
                 </li>
                 <li>
-                    <input class="w3-round w3-margin-right w-50 w3-margin-bottom" type="password" id="newpw1">
+                    <InputFile id="idProfilePicture" class="form-control" OnChange="(e)=> _editUser.ProfilePicture = e.File" />
                 </li>
-            }
-        </ul>
 
-        <hr />
+            </ul>
+
+            <hr />
 
-        <span>
-            <button onclick="@save" class="w3-large w3-margin-bottom w3-padding-16 btn w3-button w3-red w3-round w3-margin-bottom w3-margin-top w3-margin-left w3-margin-right">
-                Save
-            </button>
-        </span>
+            <span>
+                <button type="submit" class="w3-large w3-margin-bottom w3-padding-16 btn w3-button w3-red w3-round w3-margin-bottom w3-margin-top w3-margin-left w3-margin-right">
+                    Change
+                </button>
+            </span>
+        </EditForm>
 
     </div>
 </Animate>
 
 @code {
-    private bool showEditPassword = false;
+    private UserEditModel _editUser =new();
 
     private void save()
     {
         NavManager.NavigateTo("/accountDetails");
     }
-
-
-    private void editPassword()
+    private async Task AttempEdit(UserEditModel editUser)
     {
-        if (showEditPassword)
-            showEditPassword = false;
+        var attempt = await App.Authentication.UpdateUserAccount(_editUser, _editUser.ProfilePicture);
+        if (attempt.status)
+        {
+            await UtilityToolkit.CreateToast("Account Changed");
+        }
```

Comment:
Maybe navigateTo AccountDetails so the user can see in real time the changes and dosen't have to switch pages